docs/loading/CloudCanal.md

```diff
@@ -0,0 +1,91 @@
+# Load data using CloudCanal
+
+## Introduction
+
+CloudCanal Community Edition is a free data migration and synchronization platform published by [ClouGence Co., Ltd](https://www.cloudcanalx.com) that integrates Schema Migration, Full Data Migration, verification, Correction, and real-time Incremental Synchronization.
+CloudCanal help users build a modern data stack in a simple way.
+![image.png](../assets/3.11-1.png)
+
+## Download
+
+[CloudCanal Download Link](https://www.cloudcanalx.com)
+
+[CloudCanal Quick Start](https://www.cloudcanalx.com/us/cc-doc/quick/quick_start)
+
+
+## Function Description
+
+- It is highly recommended to utilize CloudCanal version 2.2.5.0 or higher for efficient data import into StarRocks.
+- It is advisable to exercise control over the ingestion frequency when using CloudCanal to import **incremental data** into StarRocks. The default import frequency for writing data from CloudCanal to StarRocks can be adjusted using the `realFlushPauseSec` parameter, which is set to 10 seconds by default.
+- In the current community edition with a maximum memory configuration of 2GB, if DataJobs encounter OOM exceptions or significant GC pauses, it is recommended to reduce the batch size to minimize memory usage.
+  - For Full DataTask, you can adjust the `fullBatchSize` and `fullRingBufferSize` parameters.
+  - For Incremental DataTask, the `increBatchSize` and `increRingBufferSize` parameters can be adjusted accordingly.
+- Supported Source endpoints and features：
+
+  | Source Endpoints \ Feature | Schema Migration | Full Data | Incremental | Verification |
+    | --- | --- | --- | --- | --- |
+  | Oracle                     | Yes | Yes | Yes | Yes |
+  | PostgreSQL                 | Yes | Yes | Yes | Yes |
+  | Greenplum                  | Yes | Yes | No | Yes |
+  | MySQL                      | Yes | Yes | Yes | Yes |
+  | Kafka                      | No | No | Yes | No |
+  | OceanBase                  | Yes | Yes | Yes | Yes |
+  | PolarDb for MySQL          | Yes | Yes | Yes | Yes |
+  | Db2                        | Yes | Yes | Yes | Yes |
+
+## Typical example
+
+CloudCanal offers a comprehensive productized capability where users can seamlessly add DataSources and create DataJobs through a visual interface. This enables automated schema migration, full data migration, and real-time incremental synchronization. The following example demonstrates how to migrate and synchronize data from MySQL to StarRocks. The procedures are similar for data synchronization between other data sources and StarRocks.
```

Comment:
change the first sentence to CloudCanal allows users to perform operations in a visual interface.